index.html

```diff
@@ -43,6 +43,15 @@
     <meta name="msapplication-TileImage" content="<%= require('./images/manifest/icon-144x144.png') %>">
     <meta name="msapplication-TileColor" content="#4b4b4b">
     <meta name="msapplication-tap-highlight" content="no">
+
+    <!-- Load and register service worker that adds offline support -->
+    <script>
```

Comment:
After I merge, I'll wrap this in an 'if' block so that it's only included in the prod index.html